From 861acd8916654f4ee3e774016dda2e46ae0fbcf7 Mon Sep 17 00:00:00 2001
From: Sabri Ünal <yakushabb@gmail.com>
Date: Sun, 28 Jun 2026 22:00:58 +0300
Subject: [PATCH] Fix `format ‘%S’ expects argument of type wchar_t` error

> error: format ‘%S’ expects argument of type ‘wchar_t *’, but argument 4 has type ‘char *’ [-Werror=format=]

Signed-off-by: Sabri Ünal <yakushabb@gmail.com>
---
 gitg/commit/gitg-commit-dialog.vala | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gitg/commit/gitg-commit-dialog.vala b/gitg/commit/gitg-commit-dialog.vala
index cf57020..9e36e72 100644
--- a/gitg/commit/gitg-commit-dialog.vala
+++ b/gitg/commit/gitg-commit-dialog.vala
@@ -939,7 +939,7 @@ class Dialog : Gtk.Dialog
 							try {
 								file.delete_async.end (res);
 							} catch (Error e) {
-								warning ("Error deleting %S file: %s", PREPARE_COMMIT_MSG_FILENAME, e.message);
+								warning ("Error deleting %s file: %s", PREPARE_COMMIT_MSG_FILENAME, e.message);
 							}
 						});
 					}
--
libgit2 1.7.2

